.github/workflows/check-commit-message.yaml

```diff
@@ -0,0 +1,44 @@
+name: 'Commit Message Check'
```

Comment:
Don't quote this value.

Note: Given that this is a YAML project, I'd like to use YAML best style practices in our supporting YAML files.
Eventually we need a yamlfmt that does what we want. (Suggestions welcome).